pkg/apis/volumesnapshot/register.go

```diff
@@ -0,0 +1,6 @@
+package volumesnapshot
+
+const (
+	// GroupName to hold the string name for the cdi uploads
+	GroupName = "snapshot.storage.k8s.io"
```

Comment:
Why are we not just vendoring the API in from https://github.com/kubernetes-csi/external-snapshotter?

Reply:
We can try that, but just thought that these types should be in the client-go later on any way. So instead of relying on another dependency, we can just copy them for now and remove once added to client-go. What do you think?

Reply:
Much cleaner to vendor this stuff in.  Also be easier to get rid of later.

Reply:
Done